src/main/java/Lottos.java

```diff
@@ -24,4 +24,12 @@ public Lotto findByOrder(int order) {
     public List<Lotto> get() {
         return this.value;
     }
+
+    public WinningResults draw(Lottos lottos, LottoNumbers winningNumbers, LottoNumber bonusNumber) {
+        List<WinningResult> winningResultList = lottos.get().stream()
+                .map(i -> new WinningResult(i, winningNumbers, bonusNumber))
+                .collect(Collectors.toList());
+
+        return new WinningResults(winningResultList, bonusNumber);
+    }
```

Comment:
```suggestion
    public WinningResults draw(WinningLotto winningLotto) {
        ...
    }
```
굳이 Lottos 객체가 자기 자신을 인자로 전달할 필요가 있을까?
당첨 번호의 두 값도 WinningLotto와 같은 객체로 묶어서 하나로 관리하는 것은 어떨까?

Reply:
자기 자신을 인자로 전달하는 부분은 불필요한 부분이라고 생각되어 제거하고, 메서드 내에서 직접 참조하도록 변경했습니다 :) 

다만, WinningLotto 객체 분리가 어떤 효과일지에 대한 의문이 있습니다.
Lotto 간 당첨(같은) 번호가 몇개인지 판단하는 로직 외에는 구현할 요구사항이 없고, 이 로직은 Lotto 객체에 있어야하지 않나 라고 생각이 들어 코멘트 남겨봅니다 :)